Frontend/ui-common/src/Engineering/Helpers/Commons.js

```diff
@@ -366,15 +366,67 @@ export const getDateFromObj = function (obj){
   return  yyyy + '-' + mm + '-' + dd;
 }
 
+function getFormattedLanguage(language){
```

Comment:
@Anuragini-2506 can you please add it for Malayalam language too.

Reply:
done